tests/unit/customizations/ecs/test_ecsdeploy.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import botocore
+import mock
+
+from awscli.testutils import unittest, capture_output
+from awscli.customizations.ecs.deploy import ECSDeploy
+
+
+class TestECSDeploy(unittest.TestCase):
+
+    TEST_APPSPEC = {
+        "version": 0.0,
+        "resources": [{
+            "TestService": {
+                "type": "AWS::ECS::Service",
+                "properties": {
+                    "taskDefinition": "arn:aws:ecs:::task-definition/test:1",
+                    "loadBalancerInfo": {
+                        "containerName": "web",
+                        "containerPort": 80
+                    }
+                }
+            }
+        }]
+    }
+
+    def setUp(self):
+        self.session = mock.Mock(spec=botocore.session.Session)
+        self.ecs_deployer = ECSDeploy(self.session)
+        self.ecs_deployer.task_def_arn = \
+            'arn:aws:ecs:::task-definition/test:3'
+        self.ecs_deployer.resources = {
+            'service': 'testService',
+            'service_arn': 'arn:aws:ecs:::service/testService',
+            'cluster': 'testCluster',
+            'cluster_arn': 'arn:aws:ecs:::custer/testCluster',
+            'app_name': 'testApp',
+            'deployment_group_name': 'testDeploymentGroup'
+        }
+
+    def test_create_and_wait_for_deployment(self):
```

Comment:
It seems like this functionality is already covered by the functional test.  I would suggest just removing this file entirely.  Also, this is using an internal method, `_create_and_wait_for_deployment`, which we typically don't use in tests.

Reply:
Codecov runs both unit and functional tests, so I'm not sure why codecov would complain. Sometimes codecov can be a little wonky. Let's try removing this again and see what codecov comes back with.

Reply:
@joguSD @jamesls  Done.